docs/extras/integrations/llms/fireworks.ipynb

```diff
@@ -75,11 +75,12 @@
     "    *   `accounts/fireworks/models/llama-v2-13b-w8a16`\n",
```

Comment:
Lets replace this with the link to https://app.fireworks.ai/ as we don't want to update this list every time we add/remove models.